akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/DDataRememberEntitiesCoordinatorStore.scala

```diff
@@ -46,6 +46,7 @@ private[akka] final class DDataRememberEntitiesCoordinatorStore(
   private val writeMajority = WriteMajority(settings.tuningParameters.updatingStateTimeout, majorityMinCap)
 
```

Comment:
these should also be changed to Plus, so we use the same approach for all of them

Reply:
https://github.com/akka/akka/pull/31802/commits/7db9a7a777a4dc5a1d6fb9baf350b561e88aea41